baggage/baggage_test.go

```diff
@@ -124,87 +121,87 @@ func TestParsePropertyError(t *testing.T) {
 
 func TestNewKeyProperty(t *testing.T) {
 	p, err := NewKeyProperty(" ")
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
This failure will stop the reaming assertions from being run. Same comment applies for this test and a lot of other tests in this PR.